

drivers/acpi/events/evmisc.c: In function `acpi_ev_queue_notify_request':
drivers/acpi/events/evmisc.c:143: warning: too many arguments for format


---

 25-akpm/drivers/acpi/events/evmisc.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN drivers/acpi/events/evmisc.c~acpi-printk-fix drivers/acpi/events/evmisc.c
--- 25/drivers/acpi/events/evmisc.c~acpi-printk-fix	2004-04-02 22:47:49.007492056 -0800
+++ 25-akpm/drivers/acpi/events/evmisc.c	2004-04-02 22:48:16.203357656 -0800
@@ -139,7 +139,8 @@ acpi_ev_queue_notify_request (
 				acpi_notify_value_names[notify_value]));
 	}
 	else {
-		ACPI_DEBUG_PRINT ((ACPI_DB_INFO, "notify value: 0x2.2_x **Device Specific**\n",
+		ACPI_DEBUG_PRINT ((ACPI_DB_INFO,
+				"notify value: 0x%2.2x **Device Specific**\n",
 				notify_value));
 	}
 

_
